python/src/aac/persistence/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""A package dealing with AaC persistence."""
+
+import os
+
+from aac.persistence.load import get_language_context_from_state_file
+
+
+__state_file_name__ = os.path.join(os.path.dirname(__file__), "..", "state.json")
```

Comment:
Let's make this a standard python constant name. Something like:
```suggestion
ACTIVE_CONTEXT_STATE_FILE_NAME = os.path.join(os.path.dirname(__file__), "..", "state.json")
```